nautobot/extras/tests/test_api.py

```diff
@@ -619,6 +624,161 @@ def test_delete_job_result_with_permission(self):
         self.assertHttpStatus(response, status.HTTP_204_NO_CONTENT)
 
 
+class ScheduledJobTest(APITestCase):
```

Comment:
Should this be an `APIViewTestCase` so that it inherits those base test functions? As it is, this class doesn't contain any tests.

Reply:
Hah, I removed the test cases based on our conversation above, not knowing that I also have to change the base class 🤦